draft-tschofenig-rats-psa-token.md

```diff
@@ -746,6 +746,9 @@ signatures and COSE_Mac0 for MACs, as defined in the COSE specification {{STD96}
 Note, however, that the use of MAC authentication is NOT RECOMMENDED due to the associated
 infrastructure costs for key management and protocol complexities.
 
+A PSA attester MUST NOT provide attestation evidence to an unauthorized challenger.
+Failing to do so may may allow an attacker to interpose between the attester and a verifier, tricking the verifier into believing the attacker is the legitimate attester.
```

Comment:
This line is very long

Reply:
Sounds good to me

Reply:
done in [a0b78aa](https://github.com/thomas-fossati/draft-psa-token/pull/117/commits/a0b78aa9537e2f0ff2c087cbce87aa0828e18928)